.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   release:
     runs-on: ubuntu-latest
-    secrets:
+    environment: release
```

Comment:
Most secrets are still global, but we can now move them over to this environment. I didn't do that, as I don't actually know our node publish keys and such. But no changes to workflow would be required in any case.